src/platform/common/utils/localize.ts

```diff
@@ -129,30 +135,45 @@ export namespace DataScienceSurveyBanner {
 }
 export namespace DataScienceRendererExtension {
     export const installingExtension = localize(
-        'DataScienceRendererExtension.installingExtension',
+        {
+            key: 'DataScienceRendererExtension.installingExtension',
+            comment: ['{Locked="Notebook Renderers"}']
```

Comment:
I am assuming we want to keep `Notebook Renderers` as is and not change it from language to language.